pkg/kube/multicluster/secretcontroller_test.go

```diff
@@ -132,9 +134,11 @@ func Test_SecretController(t *testing.T) {
 		{update: secret0UpdateKubeconfigSame},                       // 2
 		{update: secret0AddCluster, wantAdded: "c0-1"},              // 3
 		{update: secret0DeleteCluster, wantDeleted: "c0-1"},         // 4
-		{add: secret1, wantAdded: "c1"},                             // 5
-		{delete: secret0, wantDeleted: "c0"},                        // 6
-		{delete: secret1, wantDeleted: "c1"},                        // 7
+		{update: secret0ReAddCluster, wantAdded: "c0-1"},            // 5
+		{update: secret0ReDeleteCluster, wantDeleted: "c0-1"},       // 6
+		{add: secret1, wantAdded: "c1"},                             // 7
+		{delete: secret0, wantDeleted: "c0"},                        // 8
+		{delete: secret1, wantDeleted: "c1"},                        // 9
```

Comment:
@SpecialYang Could you add some desc about each case. Could change the tests to table driven